Add a page on publicly available Radius artifacts (#947)
radius-project/docs: docs/content/community/radius-artifacts.md

Reviewer: Let's name this page `media-coverage.md`

---

> type: docs
> title: "Artifacts on Radius"
> linkTitle: "Artifacts on Radius"
> description: "Information about press, blog posts, presentation and other artifacts on Radius"
> weight: 400

Reviewer: ```suggestion
type: docs
title: "Radius press coverage and media links"
linkTitle: "Press/media coverage"
description: "Learn about Radius through press articles, blog posts, presentation and other coverage of Radius"
weight: 400
```

---

> ## Microsoft Blog posts

Reviewer: Let's make this more generic. We don't want to make it just Microsoft-specific

```suggestion
## Blog posts
```

Author: good catch!

- [Radius Launch Oct 2023 - The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/) 
- [Microsoft Open Source Blog Oct 2023 - Enabling developer collaboration with Radius](https://cloudblogs.microsoft.com/opensource/2023/10/18/enabling-developer-collaboration-with-radius/)

> - Ongoing posts at [Radius Blog](https://blog.radapp.io)

Reviewer: Let's make this a markdown table instead. Something like:

| Title | Source | Date |
|-------|--------|------|
| [The Microsoft Azure Incubations Team launches Radius, a new open application platform for the cloud](https://azure.microsoft.com/en-us/blog/the-microsoft-azure-incubations-team-launches-radius-a-new-open-application-platform-for-the-cloud/)] | Microsoft Azure Blog | Oct 16, 2023 |

> ## Press

Reviewer: Let's 
```suggestion
## Press articles
```

> - [ Microsoft launches Radius, an open-source application platform for the cloud-native era - Techcrunch Oct 18,2023](https://techcrunch.com/2023/10/18/microsoft-launches-radius-an-open-source-application-platform-for-the-cloud/) 

Reviewer: Let's make this a table as well

- [Microsoft Reimagines Modern Application Deployment With Radius Platform Forbes Oct 23,2023](https://www.forbes.com/sites/janakirammsv/2023/10/23/microsoft-reimagines-modern-application-deployment-with-radius-platform/?sh=6a78a37c76d7)
- [Microsoft’s Radius and the future of cloud-native development - InfoWorld Oct 26,2023](https://www.infoworld.com/article/3709448/microsofts-radius-and-the-future-of-cloud-native-development.html)
- [Microsoft Azure introduces Radius, an open-source development platform for multi-cloud computing - ZDNET Oct 27,2023](https://www.zdnet.com/article/microsoft-azure-introduces-radius-an-open-source-development-platform-for-multi-cloud-computing/)

> ## Videos

Reviewer: Let's move this to the second slot, as the articles page is going to be super long over time and we want the shorter videos section to stand out.

> ## Presentations

Reviewer 2: Just a suggestion, more commonly used for this type of content

```suggestion
## Videos
```

> - [Mark Russinovich at the Linux Foundation Member Summit](https://aka.ms/radius-lfms)

Reviewer: Let's make this a table as well

- [Introducing Radius - Open at Microsoft ](https://www.youtube.com/watch?v=mT_NWFnYn0A)

## Cloud Native Compute Foundation Submission

> - [Radius Sandbox submission to CNCF](https://github.com/cncf/sandbox/issues/65)

Reviewer: This link probably doesn't make sense here. I'd remove it
